Add stickyness to the submit button in the staffing tab

antoinehq opened the pull request:
This PR adds stickyness to the submit row on the consultant staffing tab.

It makes this row always visible at the bottom of the screen.

<img width="1034" alt="Capture d’écran 2025-01-10 à 15 53 07" src="https://github.com/user-attachments/assets/1596fc3d-ea68-4ce6-8812-6afb29b255b9" />

digitalfox commented:
nice ! Thanks. Works great on desktop (and should be added as well to mission_staffing.html). 

Nevertheless, mobile experience is not great (at least with firefox dev tools mobile preview): 
* in vertical mode, submit buttons overlap "new staffing" rows and is not sticky
* in horizontal mode, sticky is working but partially overlap buttons and bottom margin/padding is too important

antoinehq commented:
> nice ! Thanks. Works great on desktop (and should be added as well to mission_staffing.html).
> 
> Nevertheless, mobile experience is not great (at least with firefox dev tools mobile preview):
> 
> * in vertical mode, submit buttons overlap "new staffing" rows and is not sticky
> * in horizontal mode, sticky is working but partially overlap buttons and bottom margin/padding is too important

Added on misson_staffing.html.
Fixed the overlap issue and added a border-right on mobile to "close" the submit-box as I found it cleaner not to widen it to the whole content but just the screen.  

Met an issue on Firefox Devtools where I had to activate/deactivate this particular CSS (the border) to make it appear while it was already there on Chrome Devtools... Not sure what is the source of that...

digitalfox commented:
Thanks a lot.
